planning/behavior_path_planner/src/scene_module/avoidance/avoidance_module.cpp

```diff
@@ -573,9 +576,9 @@ AvoidPointArray AvoidanceModule::calcRawShiftPointsFromObjects(
     AvoidPoint ap_return;
     ap_return.length = 0.0;
     ap_return.start_length = shift_length;
-    ap_return.start_longitudinal = o.longitudinal;
-    ap_return.end_longitudinal =
-      o.longitudinal + std::min(nominal_return_distance, return_remaining_distance);
+    ap_return.start_longitudinal = o.longitudinal + o.distance_add_to_returnpoint;
```

Comment:
@shulanbushangshu 
why do you add longitudinal distance to start _longitudinal?